app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -76,6 +77,14 @@ const NumericalInput = styled(BigNumberInput)`
   }
 `
 
+const NumericalInputError = styled.div<{ error?: string }>`
+  border-color: ${props => (props.error ? props.theme.colors.error : 'display: none')};
+  border-style: ${props => (props.error ? props.theme.textfield.borderStyle : 'display: none')};
+  border-width: ${props => (props.error ? props.theme.textfield.borderWidth : 'display: none')};
+  border-radius: ${props => (props.error ? props.theme.textfield.borderRadius : 'display: none')};
+  border: ${props => (props.error ? 'display: flow-root' : 'display: none')};
+`
+
```

Comment:
You don't need this styled component its just creating you issue as you see in image below it has some weird border. You need to change just border color property conditionally in Numerical input.
![Screenshot 2021-03-16 at 14 21 53](https://user-images.githubusercontent.com/33226956/111317158-2b863780-8664-11eb-952f-8d52193188f4.png)

Reply:
good catch. i tend to overcomplicate things, this is a much more elegant approach.